dts/bindings/sdhc/sdhc.yaml

```diff
@@ -59,3 +59,13 @@ properties:
     type: boolean
     description: |
       The host controller supports HS400 at 1.8V
+
+  mmc-dw-8bit:
+    type: boolean
+    description: |
+      The host controller supports 8bit data width
+
+  mmc-dw-4bit:
+    type: boolean
```

Comment:
do these need to be two separate properties? and does it need to be specific to mmc?

Reply:
When Driver change to higher bus speed mode (HS200, HS400 etc.) , the bus width also need to change and based on host capabilities (whether it support 8bit or not etc.) it can decide whether it can change higher speed. So it need to be separate.

Reply:
okay, what I meant is would it better to combine them to a single property that was a description of the bus width like linux does

Reply:
I am not sure how or whether we can use it to determine the specific capabilities supported. 

https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/mmc/mmc-controller.yaml
  bus-width:
    description:
      Number of data lines.
    $ref: /schemas/types.yaml#/definitions/uint32
    enum: [1, 4, 8]
    default: 1

Reply:
why not? does it not work for linux? If I remember correctly, the rest of the binding properties we tried to align to linux